app/routes/index.tsx

```diff
@@ -110,7 +110,7 @@ export default function App() {
 
   const handleSpecialLinks = (e: MouseEvent) => {
     const el = e.target as HTMLAnchorElement
-    if (el.tagName !== 'A') return
+    if (el.tagName !== 'A' || el.closest('.question-footer')) return
```

Comment:
Dunno about this. I was thinking of introducing some kind of constant or something, in case the class name changes

Reply:
probably best if the <a> element has some special class used only for this purpose.. but whatever works is fine I guess